middlewares/checkCanGenerateDiscordLink.ts

```diff
@@ -7,7 +7,7 @@ const checkCanGenerateDiscordLink = async (req: CustomRequest, res: CustomRespon
   const isSuperUser = roles.super_user;
   const userIdInQuery = req.query.userId;
   const currentTime = Date.now();
-  const cutoffTime = 1724630399000;  // Epoch time for 25 August 2024
+  const cutoffTime = 1725147849000;  // Todo will remove this Hotfix time for 31 August 2024
```

Comment:
NIT: @vinit717 Please use a timestamp like 11:59pm next time (It's okay for now, because we are just extracting the date part and this becomes Sunday, September 1, 2024 5:14:09 AM IST)